projects/demo/src/modules/components/items-with-more/items-with-more.template.html

```diff
@@ -0,0 +1,111 @@
+<tui-doc-page
+    header="ItemsWithMore"
+    package="KIT"
+    type="components"
+>
+    <ng-template pageTab>
+        <div
+            i18n
+            class="tui-space_bottom-3"
+        >
+            Component to hide overflown items behind custom content.
+        </div>
+
+        <tui-doc-example
+            id="base"
+            i18n-heading
+            heading="Basic"
+            [content]="example1"
+        >
+            <tui-notification class="tui-space_bottom-3">
+                Resize the screen to see extra items disappear
+            </tui-notification>
+            <tui-items-with-more-example-1></tui-items-with-more-example-1>
+        </tui-doc-example>
+
+        <tui-doc-example
+            id="dropdown"
+            i18n-heading
+            heading="Dropdown"
+            [content]="example2"
+        >
+            <tui-items-with-more-example-2></tui-items-with-more-example-2>
+        </tui-doc-example>
+    </ng-template>
+
+    <ng-template pageTab>
+        <tui-doc-demo>
+            <tui-items-with-more
+                [itemsLimit]="itemsLimit"
+                [required]="required"
+            >
+                <ng-container *ngFor="let item of items">
+                    <tui-tag
+                        *tuiItem
+                        class="tui-space_right-2"
+                        [value]="item"
+                    ></tui-tag>
+                </ng-container>
+                <span *tuiMore>and now for something completely different!</span>
+            </tui-items-with-more>
+        </tui-doc-demo>
+        <p>
+            Use
+            <code>tuiItem</code>
+            directive for each item
+        </p>
+        <p>
+            Use
+            <code>tuiMore</code>
+            directive for "See more" content
+        </p>
+        <tui-doc-documentation>
+            <ng-template
+                i18n
+                documentationPropertyName="itemsLimit"
+                documentationPropertyMode="input"
+                documentationPropertyType="number"
+                [documentationPropertyValues]="itemsLimitVariants"
+                [(documentationPropertyValue)]="itemsLimit"
+            >
+                Artificial limit on visible items
+            </ng-template>
+            <ng-template
+                i18n
+                documentationPropertyName="required"
+                documentationPropertyMode="input"
+                documentationPropertyType="number"
+                [documentationPropertyValues]="requiredVariants"
+                [(documentationPropertyValue)]="required"
+            >
+                Index of an item that must remain visible
```

Comment:
nit:

Should we add possibility to set multiple indexes (`number` => `number[]`) ?

Reply:
I would keep it single for now, it might overcomplicate things.